README.md

```diff
@@ -53,7 +53,7 @@ const mySchema = [
   },
 ];
 
-let myERC725 = new ERC725(schema[, address, provider, ipfsGateway])
```

Comment:
i guess we want to add here an example config object and also use the variables from above.

best move line 28 and 27 below 54 you

Reply:
Yes, very good input.